samples/browser/src/geckoNightly/java/org/mozilla/samples/browser/Components.kt

```diff
@@ -20,6 +20,6 @@ class Components(private val applicationContext: Context) : DefaultComponents(ap
         }
 
         val runtime = GeckoRuntime.getDefault(applicationContext)
-        GeckoEngine(runtime, defaultSettings)
+        GeckoEngine(applicationContext, runtime, defaultSettings)
```

Comment:
Since we now no longer need to specify the runtime, we have two options:

```Kotlin
GeckoEngine(context = applicationContext, defaultSettings = defaulSettings)
```

or we could change the parameter ordering:

```Kotlin
GeckoEngine(applicationContext, defaultSettings)
```

Reply:
Changed